assets/prefabs/food/raw/Grapefruit.prefab

```diff
@@ -0,0 +1,28 @@
+Grapefruit
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:Grapefruit",

```

Comment:
The name "Grapefruit" here should match the actual image file name. So you would want to rename the image file from `grape fruit.png` to `Grapefruit.png`